matchmaker/models.py

```diff
@@ -41,6 +47,10 @@ class MatchmakerIncomingQuery(ModelWithGUID):
     def __unicode__(self):
         return '{}_{}_query'.format(self.patient_id or self.id, self.institution)
 
+    def __str__(self):
+        """Magic function for 'str()' and '%s'."""
+        return '{}_{}_query'.format(self.patient_id or self.id, self.institution)
```

Comment:
for all of these, is there a reason `__str__` can't just return the same thing as `__unicode__`?
So for example, 
```
def __str__(self):
   return self.__unicode__()
```

If that would work, you could add it to the base class instead of each individual class

Reply:
Yes. It will work.

Reply:
Moved to the base class.